test/ContentBase.test.js

```diff
@@ -256,50 +256,59 @@ describe('ContentBase', () => {
   });
 
   describe('testing single & multiple external paths', () => {
+    afterAll((done) => {
+      testServer.close(() => {
+        done();
+      });
+    });
     it('Should throw exception (string)', (done) => {
       try {
         // eslint-disable-next-line no-unused-vars
         server = testServer.start(config, {
+          port: 9004,
           contentBase: 'https://example.com/',
           watchContentBase: true,
         });
 
         expect(true).toBe(false);
       } catch (e) {
         expect(e.message).toBe('Watching remote files is not supported.');
-        testServer.close(() => {
-          done();
-        });
+        done();
       }
     });
     it('Should throw exception (array)', (done) => {
       try {
         // eslint-disable-next-line no-unused-vars
         server = testServer.start(config, {
+          port: 9003,
           contentBase: [contentBasePublic, 'https://example.com/'],
           watchContentBase: true,
         });
 
         expect(true).toBe(false);
       } catch (e) {
         expect(e.message).toBe('Watching remote files is not supported.');
-        testServer.close(() => {
-          done();
-        });
+        done();
       }
     });
   });
 
   describe('testing single & multiple internal paths', () => {
```

Comment:
tests in this part are not necessary, all other tests already test with valid contentBase.